backend/src/tasks/cve.ts

```diff
@@ -30,8 +34,14 @@ export const handler = async (commandOptions: CommandOptions) => {
           product.cpe &&
           product.version &&
           product.version.split('.').length > 1
-        )
+        ) {
           cpes.add(product.cpe + ':' + product.version);
+          if (productMap[product.cpe]) {
+            for (const cpe in productMap[product.cpe]) {
+              cpes.add(cpe + ':' + product.version);
```

Comment:
I don't think .NET framework and ASP.NET versions necessarily coincide. See https://www.tutorialsteacher.com/mvc/asp.net-mvc-version-history

Reply:
Actually, maybe they do, at least with later versions -- ASP.NET Core seems to be the same thing as .NET -- see https://github.com/dotnet/aspnetcore